src/main.cpp

```diff
@@ -542,19 +582,26 @@ int main(int argc, char**argv)
   seed = time(NULL) ^ getpid();
   srand(seed);
 
+  // Create a DNS resolver.
+  int af = AF_INET;
+  struct in6_addr dummy_addr;
+  if (inet_pton(AF_INET6, options.local_host.c_str(), &dummy_addr) == 1)
+  {
+    TRC_DEBUG("Local host is an IPv6 address");
+    af = AF_INET6;
+  }
+
+  DnsCachedResolver* dns_resolver = new DnsCachedResolver("127.0.0.1");
```

Comment:
I thought that memento didn't have split-network?

Also, I just moved code up a few lines so that it came before the AstaireResolver.

I may as well use --dns-servers though.